test/Client/OrchestrationServiceClientShim.Tests/ShimDurableTaskClientTests.cs

```diff
@@ -174,6 +174,7 @@ public async Task PurgeInstanceMetadata()
         // assert
         this.orchestrationClient.VerifyAll();
         result.PurgedInstanceCount.Should().Be(1);
+        result.IsComplete.Should().BeNull();

```

Comment:
I would change this to assert that `IsComplete` should not be false. It's correct for it to be either null or true, depending on the backend implementation.